eessi-2023.06-eb-4.7.2-2022b.yml

```diff
@@ -5,3 +5,6 @@ easyconfigs:
       options:
         include-easyblocks-from-pr: 2248
   - foss-2022b
+  - WRF-4.4.1-foss-2022b-dmpar.eb:
+      options:
+        include-easyblocks-from-pr: 3012
```

Comment:
We should add a comment here to mention why this is needed, and ideally we add a separate entry for `netCDF` and only use this there.

I'll look into that and update the PR, since the builds are already done.

Reply:
fixed in c0677ad